### PR TITLE
RavenDB-8518 - PR comments fixes

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

File: src/Raven.Studio/typescript/common/notifications/models/abstractNotification.ts [hunk 1]
```diff
@@ -12,6 +12,8 @@ abstract class abstractNotification {
     title = ko.observable<string>();
     severity = ko.observable<Raven.Server.NotificationCenter.Notifications.NotificationSeverity>();
     type: Raven.Server.NotificationCenter.Notifications.NotificationType;
+    licenseLimitType = ko.observable<Raven.Server.Commercial.LimitType>();
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/notifications/models/abstractNotification.ts:15]: this field is calulated based on details - I think we shouln't introduce field used only in one case on *every* notification.

File: src/Raven.Studio/typescript/common/notifications/models/alert.ts [hunk 2]
```diff
@@ -27,6 +27,11 @@ class alert extends abstractNotification {
         this.alertType(incomingChanges.AlertType);
         this.key(incomingChanges.Key);
         this.details(incomingChanges.Details);
+        if (incomingChanges.AlertType === "LicenseManager_LicenseLimit") {
```
Comment (ml054) [inline src/Raven.Studio/typescript/common/notifications/models/alert.ts:30]: then we can remove this

File: src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/licenseLimitDetails.ts [hunk 3]
```diff
@@ -12,27 +12,38 @@ class licenseLimitDetails extends dialogViewModelBase {
...
         this.licenseLimitNotification = licenseLimitNotification;
 
-        this.url = ko.pureComputed(() => {
-            let url = "https://ravendb.net/request-license?";
-            const error = licenseLimitNotification as recentError;
-            if (error && error.licenseLimitType()) {
-                url += `limit=${error.licenseLimitType()}`;
+        this.requestLicenseUrl = ko.pureComputed(() => {
+            let url = `${license.baseUrl}?`;
+            const limitType = licenseLimitNotification.licenseLimitType();
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/licenseLimitDetails.ts]: and compute this here - based on details

File: src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/licenseLimitDetails.ts [hunk 4]
```diff
@@ -12,27 +12,38 @@ class licenseLimitDetails extends dialogViewModelBase {
...
+            if (licenseLimitNotification && limitType) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/common/notificationCenter/detailViewer/licenseLimitDetails.ts]: licenseLimitNotification  this is not needed here

File: src/Raven.Studio/typescript/viewmodels/shell/registration.ts [hunk 5]
```diff
@@ -68,41 +68,43 @@ class registration extends dialogViewModelBase {
...
         this.daysToRegister = ko.pureComputed(() => {
             const now = moment();
             return firstStart.diff(now, "days");
         });
 
-        let url = "https://ravendb.net/request-license?";
-        if (license && license.Id) {
-            url += `&id=${license.Id}`;
+        let url = license.baseUrl;
+        if (licenseStatus && licenseStatus.Id) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/shell/registration.ts]: code duplication. I think computing this should be done in single place.

File: src/Raven.Studio/typescript/common/notifications/models/abstractNotification.ts (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Studio/typescript/common/notifications/models/abstractNotification.ts:15]: Computed field?

**Pull request merged**